regions_www/m/pct_shiny_funs.R

```diff
@@ -164,9 +164,9 @@ data_filter <- function(scenario, type){
   ifelse(scenario == "olc", "bicycle", paste(scenario, type, sep = "_"))
 }
 
-# Create a new variable called font_colour which changes into red colour when change in death/yr is negative
-positive_red <- function(dataset, scenario, type){
-  dataset@data$font_colour <- ifelse(round(dataset[[data_filter(scenario, type)]]) <0, "red", "black")
+# Return red if the data is positive
```

Comment:
yes, nice, except that it is actually red if the data is negative not positive – I have renamed function accordingly in one of my commits